core/src/state.rs

```diff
@@ -93,6 +93,13 @@ impl State {
             holochain_public_key,
         ))
     }
+
+    pub fn get_admin_public_key(&self) -> Result<PublicKey, Error> {
+        match self {
+            State::V1{seed: _, config: c} => Ok(c.admin.public_key),
+            _ => panic!("Wrong version of hpos_state file. Expecting v1.")
```

Comment:
This can never panic, because you've enumerated all possible state versions above. Instead, this should be:

```suggestion
            _ => unreachable!()
```